worker/Dockerfile.template

```diff
@@ -34,9 +34,7 @@ RUN apk add --no-cache \
   libusb-dev dbus-dev eudev-dev \
   gstreamer-tools gst-plugins-base gst-plugins-bad gst-plugins-good \
   bridge bridge-utils iproute2 dnsmasq iptables \
-  qemu-img qemu-system-x86_64 \
-    && rm /usr/share/qemu/edk2* `# remove huge edk2 firmware` \
-  && (apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/v3.12/community ovmf || true)
+  qemu-img qemu-system-x86_64 qemu-system-aarch64
```

Comment:
I checked the contents of these packages on the alpine repos and none of them include the needed code or vars files.
- https://pkgs.alpinelinux.org/contents?file=&path=&name=qemu-img&branch=v3.12&repo=community&arch=x86_64
- https://pkgs.alpinelinux.org/contents?branch=v3.12&name=qemu-system-x86_64&arch=x86_64&repo=community
- https://pkgs.alpinelinux.org/contents?file=&path=&name=qemu-system-aarch64&branch=v3.12&repo=community&arch=x86_64